content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -124,7 +124,7 @@ different Kubernetes components.
 | `HugePageStorageMediumSize` | `false` | Alpha | 1.18 | 1.18 |
 | `HugePageStorageMediumSize` | `true` | Beta | 1.19 | |
 | `IndexedJob` | `false` | Alpha | 1.21 | |
-| `IngressClassNamespacedParams` | `false` | Alpha | 1.21 | |
```

Comment:
please leave this line in place and add the `Beta` version below it.

Reply:
Updated!